meson.build

```diff
@@ -218,8 +218,11 @@ if build_arping == true
 	endif
 endif
 
-if build_mans == true or build_html_mans == true
-	subdir ('doc')
+if meson.version().version_compare('>=0.58.0')
+	if build_mans == true or build_html_mans == true
+		subdir ('doc')
+		meson.add_dist_script('meson-build-dist-man.sh')
+	endif
```

Comment:
Typo for 0.48 (when dist scripts were added)?

Anyway I think you don't want to skip the entire doc/ directory on too old versions of meson.

Reply:
Yeah, I think I was copying the 0.58.0 from `doc`. Good catch on the skipping...